.github/workflows/coverage.yml

```diff
@@ -132,10 +132,10 @@ jobs:
           timeout: 10000
           moonbeam-binary: build/moonbeam
           force-pass: true
-      - name: "Run Moonwall Dev Tests"
-        uses: ./.github/workflow-templates/dev-tests
-        with:
-          moonwall_environment: dev_moonbase
+      # - name: "Run Moonwall Dev Tests"
+      #   uses: ./.github/workflow-templates/dev-tests
+      #   with:
+      #     moonwall_environment: dev_moonbase
```

Comment:
Is this intentional? If so, I would remove it rather than comment it out.

Reply:
Oh, is it related to this?

> Also, we are disabling code coverage for TypeScript-based tests due to the slow launch of the node, which has caused it to be broken.

Reply:
Yes, I disabled code coverage for TypeScript-based tests because the slow launch of the node was causing them to fail. This is temporary, and the check should be re-enabled once the problem is resolved.